Mailbox.Tests/DataLoaderTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+    public class DataLoaderTests
+    {
+        [TestMethod]
+        public void Load_OneMailbox_ReturnsCorrectValue()
+        {
+            //Arrange
+            Mailbox mailbox = new Mailbox(Sizes.Small, (0, 0), new Person("Homer", "Simpson"));
+            MemoryStream memoryStream = new MemoryStream();
+            using (StreamWriter streamWriter = new StreamWriter(memoryStream, new UTF8Encoding(), 1024, true))
```

Comment:
There are static helpers for getting access to the encodings.
```suggestion
            using (StreamWriter streamWriter = new StreamWriter(memoryStream, System.Text.Encoding.UTF8, 1024, true))
```